docs/logging-handlers.rst

```diff
@@ -0,0 +1,7 @@
+Logger
+======
```

Comment:
This title, which appears in side-navigation, should give the user a clue that the API is about integration with the Python `logging` module.